src/model/cards/legends/bridge_of_the_sky.ts

```diff
@@ -0,0 +1,37 @@
+import { Card } from "../../card";
+import { GameState } from "../../gameState";
+import { toGameText } from "../../gameText";
+import {
+  CardName,
+  CardType,
+  ExpansionType,
+  GameInput,
+  ResourceType,
+} from "../../types";
+
+export const bridge_of_the_sky: ConstructorParameters<typeof Card>[0] = {
+  expansion: ExpansionType.LEGENDS,
+  name: CardName.BRIDGE_OF_THE_SKY,
+  associatedCard: CardName.ARCHITECT,
+  upgradeableCard: CardName.CRANE,
+  cardType: CardType.GOVERNANCE,
+  cardDescription: toGameText([
+    "You may play 1 ",
+    { type: "em", text: "Construction" },
+    "for -3 ANY, then place it on top of this card.",
+    "PLUS is equal to the POINT value of the ",
```

Comment:
Would use the horizontal line break to separate the point values similar to what we do events.